src/components/share-overlay/share-overlay.js

```diff
@@ -150,12 +176,43 @@ const VideoStartOptions = (props: Object): React$Element<any> => {
    * @memberof VideoStartOptions
    */
   const onKeyDown = (e: KeyboardEvent): void => {
-    if (e.keyCode === KeyMap.ENTER) {
+    if ([KeyMap.ENTER, KeyMap.SPACE].includes(e.keyCode)) {
       e.preventDefault();
       props.toggleStartFrom();
     }
   };
 
+  /**
+   * on change handler
+   * prevent invalid chars in input (only numbers and ':' are valid)
+   * @param {string} val - the value from the input element
+   * @returns {void}
+   * @memberof VideoStartOptions
+   */
+  const onInputChangeHandler = (val: string): void => {
+    for (let index = 0; index < val.length; index++) {
+      const char = val.charAt(index);
+      if (!/\d/.test(char) && char !== ':') {
+        _inputRefElement.value = val.replace(char, '');
+        break;
+      }
+    }
+  };
+
+  const inputProps = {
+    'aria-labelledby': 'start-from-label',
+    'aria-disabled': props.startFrom ? 'false' : 'true',
+    type: 'text',
+    value: toHHMMSS(props.startFromValue),
+    className: [style.formControl, shareStyle.startAtInput, !props.startFrom ? shareStyle.disabled : ''].join(' '),
+    style: props.videoHasHours ? 'width: 85px;' : 'width: 56px;',
```

Comment:
can this be moved into the css file ?

Reply:
i already tried to move it to the css file, but the style is being overridden by another class named .playkit-player .playkit-form-control. that made me think that this is why it was as inline at first place.

Reply:
ok